app/assets/javascripts/pretty-text/upload-short-url.js.es6

```diff
@@ -53,8 +53,11 @@ function _loadCachedShortUrls($uploads) {
 
           if (url !== MISSING) {
             $upload.attr("href", url);
-            const content = $upload.text().split("|");
 
+            // Replace "|attachment" with class='attachment'
+            // TODO: This is a part of the cooking process now and should be
+            // removed in the future.
+            const content = $upload.text().split("|");
```

Comment:
if there's any chance `$upload.text()` can be `""` we should do:

```suggestion
            const content = $upload.text().split("|").filter(Boolean);
```

Reply:
`[double pipe ||]()` should generate a link with title 'double pipe ||'. If blank values would be filtered, then the output would be 'double pipe '.